iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/RoutePreview/MWMRoutePointCell.h

```diff
@@ -0,0 +1,26 @@
+//
+//  MWMRoutePointCell.h
+//  Maps
+//
+//  Created by v.mikhaylenko on 22.09.15.
+//  Copyright © 2015 MapsWithMe. All rights reserved.
```

Comment:
Remove them (and fix XCode tempate if necessary to avoid autogenerated headers in the future).